### PR TITLE
[E2E] Clean up the E2E with Java API to avoid platform sensitive

### PR DESCRIPTION
**Pull request by kezhenxu94**
And enhance the docker-compose file to make it more stable

### CODE REVIEW

**Comment by kezhenxu94** [comment]
FYI @SbloodyS

File: dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/java/org/apache/dolphinscheduler/e2e/cases/FileManageE2ETest.java [hunk 1]
```diff
@@ -95,18 +98,14 @@ public static void setup() {
...
-            int status = pro.waitFor();
-            if (status != 0) {
-                throw new RuntimeException(String.format("Failed to call shell's command: %s", Arrays.toString(command)));
-            }
-        } catch (Exception e) {
-            throw new RuntimeException(e);
-        }
+        Files.deleteIfExists(testUnder1GBFilePath);
+        Files.deleteIfExists(testOver1GBFilePath);
+        Files.walk(Constants.HOST_CHROME_DOWNLOAD_PATH)
```
Comment (SbloodyS) [inline dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/java/org/apache/dolphinscheduler/e2e/cases/FileManageE2ETest.java:105]: I have a question. The folder deleted in this place is a folder created by root user. Why is there no permission exception? I tried this method before, but threw an exception with insufficient permission.

**Review by zhongjiajie — APPROVED** [review]
LGTM

**Comment by zhongjiajie** [comment]
@kezhenxu94 I delete the branch `cleanup/e2e` unintentionally, restores it if you still need it. BTW, @SbloodyS add comment in https://github.com/apache/dolphinscheduler/pull/8197#discussion_r792313788

**Comment by kezhenxu94** [comment]
> @kezhenxu94 I delete the branch `cleanup/e2e` unintentionally, restores it if you still need it.

We should always delete branches after PR merged, thanks.

> BTW, @SbloodyS add comment in [#8197 (comment)](https://github.com/apache/dolphinscheduler/pull/8197#discussion_r792313788)

Will take a look

**Pull request merged**